src/core_plugins/kibana/public/visualize/editor/editor.html

```diff
@@ -80,6 +80,9 @@
     <div class="vis-editor-canvas" ng-class="{ embedded: !chrome.getVisible() }">
       <visualize
         vis="vis"
+        reporting-item
```

Comment:
assuming this will only be used for reporting the names are great ... but if we are looking forward to standard api, should we try to find more general name for this ?